openedx/core/djangoapps/programs/utils.py

```diff
@@ -222,16 +222,16 @@ def progress(self, programs=None, count_only=True):
             completed, in_progress, not_started = [], [], []
 
             for course in program_copy['courses']:
-                entitlement = CourseEntitlement.objects.filter(user=self.user,
-                                                               course_uuid=course['uuid']).first()
-
+                active_entitlement = CourseEntitlement.get_entitlement_if_active(user=self.user,
```

Comment:
Nit: You should look at the reference above for this line also.  I am surprised this does not throw "quality" linter warnings.

Reference: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines